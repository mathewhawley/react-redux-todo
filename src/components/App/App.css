@import '../../styles/variables.css';

.main {
  margin-top: 60px;
}

@media (--small-up) {
  .main {
    margin-top: 70px;
  }
}
