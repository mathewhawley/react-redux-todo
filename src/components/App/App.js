import React from 'react';
import { TodoFormContainer } from '../TodoForm/TodoFormContainer';
import { TodoListContainer } from '../TodoList/TodoListContainer';
import { Banner } from '../Banner';
import styles from './App.css';

const App = () => {
  return (
    <div>
      <Banner />
      <main className={styles.main}>
        <TodoFormContainer />
        <TodoListContainer />
      </main>
    </div>
  );
};

App.displayName = 'App';

export default App;
