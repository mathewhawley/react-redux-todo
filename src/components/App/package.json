{
  "private": true,
  "name": "App",
  "main": "./App"
}
