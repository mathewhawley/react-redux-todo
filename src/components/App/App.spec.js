/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';

import { App } from './App';

describe('<App />', () => {
  describe('Snapshots', () => {
    it('should render as expected', () => {
      const component = renderer.create(<App />);
      const tree = component.toJSON();

      expect(
        tree
      ).toMatchSnapshot();
    });
  });
});
