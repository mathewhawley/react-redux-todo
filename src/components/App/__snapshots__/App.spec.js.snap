exports[`<App /> Snapshots should render as expected 1`] = `
<h1>
  Hello, world
</h1>
`;
